### DISCUSSION TITLE
什麼是web3.0

### DISCUSSION THREAD

**Discussion by yuhao112** [1101-GS4719-C/Python-Community-GS4719-C#521] [Show and tell]
https://www.blocktempo.com/what-is-web-3-let-us-explain-using-simple-terms/
最近很流行一個詞彙，那就是web3.0。對於剛發展的階段，對很多人來說確實是一個很陌生的名詞，簡單來說是屬於一種全新網路的模式，文章由提及到，從web1.0的讀訊息到web2.0的讀訊息再加上書寫訊息再到現在web3.0甚至能擁有訊息，網路的發展其實是很迅速地。然而多數人到目前始終不認得這項技術，超過80%的美國人依舊認為只是個騙局而已，然而20年前當時的人連網路這個詞都很陌生，現在卻是人人都在使用，發展得如此迅速，相信在未來的某一天，web3.0的應用肯定也是能在生活四周的!

**Comment by boson13579** [comment]
Web3.0是internet future。從傳訊息給朋友的方式，到現在能keep the message。它用到了區塊鏈和去中心化技術，讓我們可以擁有真正的數位資產所有權。這將有可能影響金融或遊戲和社交等領域。儘管我們對Web3.0還不熟悉，但就像網路的發展一樣，我相信它會迅速成為生活中不可或缺的一部分!
